website/static/js/logTextParser.js

```diff
@@ -25,6 +25,12 @@ function ravenMessage (message, logObject) {
     }
 }
 
+function toRelativeUrl(url) {
+    var parser = document.createElement('a');
```

Comment:
Also, I'm unfamiliar with that pattern of URL manipulation. Is that a Mithril thing? Will it work on all our supported browsers?